pennylane/devices/default_gaussian.py

```diff
@@ -106,7 +106,7 @@ def fock_prob(cov, mu, event, hbar=2.0):
     """
     # number of modes
     N = len(mu) // 2
-    I = np.identity(N)
+    id = np.identity(N)
```

Comment:
@rmoyard `id` is a python intrinsic function, so we shouldn't use it for variable assignment 🙂

Reply:
I partially wonder if we should suppress this warning, since I find `I` much more readable, and the distinction between I and 1 is okay in fixed-width fonts 🤔